CODE_OF_CONDUCT.md

```diff
@@ -45,9 +45,9 @@ threatening, offensive, or harmful.
 
 ## Scope
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
+This Code of Conduct applies both within project spaces and in public spaces.
+When an individual is representing the project or its community. Examples of
+representing a project or community includes using an official project e-mail
```

Comment:
The `includes` change makes sense to me, but the splitting of the sentence doesn't work since it's a continuation of the same sentence.
```suggestion
This Code of Conduct applies both within project spaces and in public spaces
when an individual is representing the project or its community. Examples of
representing a project or community includes using an official project e-mail
```